.github/workflows/reproducibility.yaml

```diff
@@ -58,6 +77,15 @@ jobs:
         run: |
           ./scripts/docker_run ./scripts/xtask build-oak-functions-server-variants
 
+      # Upload artifacts to Ent, where they will be retained and publicly accessible by their hash.
+      # This only applies to `push` events, since it needs access to a valid Ent API key (which is
+      # not available on forks / PRs).
+      - name: Upload to Ent
+        if: github.event_name == 'push'
+        run: |
+          ent put ./target/x86_64-unknown-linux-musl/release/oak_functions_loader_base
+          ent put ./target/x86_64-unknown-linux-musl/release/oak_functions_loader_unsafe
```

Comment:
Could something be printed out in the next step listing the URIs of the uploaded files to `ent`?

Reply:
In fact this already prints something: https://github.com/project-oak/oak/runs/6023254832?check_suite_focus=true#step:9:5

I guess we could use this instead of / in addition to the repro index at some point, but probably it's too early for that.